bin/varnishd/proxy/cache_proxy_proto.c

```diff
@@ -705,11 +705,11 @@ VPX_Send_Proxy(int fd, int version, const struct sess *sp)
 	} else
 		WRONG("Wrong proxy version");
 
-	(void)write(fd, VSB_data(vsb), VSB_len(vsb));
+	r = write(fd, VSB_data(vsb), VSB_len(vsb));
```

Comment:
Forget my previous comment regarding error handling, you took care of it but simply removed the comment too soon.